jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JacksonModule.java

```diff
@@ -248,4 +252,19 @@ protected boolean shouldIgnoreField(FieldScope field) {
         return beanDescription.findProperties().stream()
                 .noneMatch(propertyDefinition -> fieldName.equals(propertyDefinition.getInternalName()));
     }
+
+    /**
+     * Look-up the given field's {@link JsonProperty} annotation and its "required" attribute,  and calculate wether or not this field should be part of the required properties
```

Comment:
two checkstyle rules complain here: the line is longer than 150 characters and a period symbol is missing to end the first sentence in the JavaDoc. Maybe just shorten it a bit:
```suggestion
     * Look-up the given field's {@link JsonProperty} annotation and consider its "required" attribute.
```